_posts/2018-03-01-core-team-announcement.md

```diff
@@ -0,0 +1,34 @@
+---
+layout: post
+title: Core Team Announcement
+summary: Small elaboration on the Core Team's role and an announcement on its membership
+tags:  [monero, core]
+author: The Core Team
+---
+
+The year 2018 is well underway, with many exciting new technologies on the horizon for Monero. Technology is not the only thing that needs to adapt if Monero is to stay at the forefront of privacy and blockchain technologies, however. With that in mind, the Core Team would like to remind the community about their role in regards to Monero followed by an announcement regarding the membership of the Core Team. 
+
+The role of the Core Team can often be a confusing topic, especially with the various leadership structures employed by other cryptocurrencies. As the technology for a completely trustless society is not yet a reality, all projects, even decentralized ones, benefit from having individuals that can carry out tasks that require trust. A quick example of this is owning a domain name (like getmonero.org) or maintaining servers (on which getmonero.org is hosted). Ultimately, there is not yet a way for a domain name to be owned by a decentralized community, so someone has to be the name holding this domain. It's in scenarios like this where the Core Team are able to work on bridging the decentralized world we are trying to bring about with the centralized reality in which we are currently living.
+
+The responsibilities of the Core Team are as follows (in no particular order):
+    - Act as primary trusted arbiters of the Forum Funding System on behalf of the community, so as to ensure the completion of all projects to the satisfaction of the community.
+    - Manage the codebase of the Monero Project, which includes merging code on Github, keeping backups, and ensuring the safety, security, and free access of the code from any party.
+    - Steward the general donation fund, and spending the Monero there on anything they see fit to further the Monero Project.
+    - Act as trusted signers and distributors of reference clients for the Monero coin, and other related technologies.
+    - Set a direction and vision for the Monero Project

```

Comment:
>Set a direction and vision for the Monero Project

This can't be done because the core team is purportedly not an authority. Setting direction and vision is the job of an authority - and centralization is a requisite for authority (see comment above).

Reply:
>"Propose a direction and vision for the Monero Project"

I think this is an improvement but the fatal flaw is that this sets the precedent for democracy in a place where votes can be generated by random identities on the fly. Also, politics implies bureaucracy, appointees, officers, authorities etc. so, not only is that dangerous (politicians are bought all the time), but that is also beyond the purview of the core team.

Another danger: since the core team holds FFS funds, if the core team *proposes* an idea, this is essentially a suggestion of direction for any contributor *after* they are funded. If the contributor disagrees with the proposal, and they are outvoted by the community (but said contributor knows better because they have experience and skill-sets of which the community, in this example, doesn't have) the outcome will negatively affect the software and also risk food on the table for the contributor.

>Personally I think it is mostly about helping to keep the early community vision when said community keeps growing.

Referencing https://github.com/monero-project/monero-site/pull/634#discussion_r172493038.